pkg/middleware/request_metrics.go

```diff
@@ -7,13 +7,32 @@ import (
 	"time"
 
 	"github.com/grafana/grafana/pkg/infra/metrics"
+	"github.com/prometheus/client_golang/prometheus"
 	"gopkg.in/macaron.v1"
 )
 
+var (
+	httpRequestsInFlight prometheus.Gauge
+)
+
+func init() {
+	httpRequestsInFlight = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "http_requests_in_flight",
```

Comment:
To follow the naming of the other http_request_ metrics I suggest the following change:

```suggestion
			Name: "http_request_in_progress",
```

Reply:
@marefr Why rename in_flight to in_progress? I think they are synonymous? Renaming from http_requests_ to http_request is to be consistent, AFAICT, even though I'm not familiar with the history.

Reply:
Personal opinion. "in flight" makes me think about flights/flying, but maybe that's only me?

Reply:
I think "in flight" is a normal term to refer to for example requests being handled. You can see [here](https://english.stackexchange.com/a/189987) for a discussion.

Reply:
Okay thanks :+1:

Reply:
Renamed to `http_request_in_flight`.